documentation/src/main/asciidoc/query.asciidoc

```diff
@@ -289,6 +289,74 @@ Query luceneQuery = mythQB.keyword()
 In the previous example, only field name is boosted to 5.
 
 
+===== Plain text queries
+
+Plain text queries use the Lucene SimpleQueryParser. The query text is parsed and it exposes more
```

Comment:
Maybe we should add something to the Elasticsearch integration doc, to mention that the syntax there is the one for `simple_query_string` (which may be slightly different, at least in the future).